winsup/cygwin/local_includes/cygtls.h

```diff
@@ -225,22 +225,21 @@ class _cygtls
   int call_signal_handler ();
   void remove_wq (DWORD);
   void fixup_after_fork ();
-#if !defined(__aarch64__)
   void lock ()
   {
     while (InterlockedExchange (&stacklock, 1))
       {
 #if defined(__x86_64__)
 	__asm__ ("pause");
+#elif defined(__aarch64__)
+	// TODO: Implement this.
```

Comment:
What else should be implemented?

Reply:
This change fixes build but do not validate whether it actually works. This TODO is for the future to validate it.

Reply:
The change looks reasonable, "validation is required" - is better comment, perhaps. this one is fine as well.

Reply:
Comment updated.